SUSEConnect.8.ronn

```diff
@@ -99,31 +99,30 @@ Manage subscriptions at [SUSE Customer Center][scc]
 ### AFTER
   `SUSEConnect --url <registration-server-url> -r <regcode> >> <logfile>`
 
-## USE WITH SMT
+## USE WITH REGISTRATION PROXY
   SUSEConnect can also be used to register systems with a local SUSE
-  Subscription Management Tool instead of the SUSE Customer Center.
-  Use `SUSEConnect --url <smt-server-url>` to register systems with SMT and set
-  [proxies][ENVIRONMENT] as required.
+  registration proxy (RMT/SMT) instead of the SUSE Customer Center.
+  Use `SUSEConnect --url <registration-proxy-server-url>` to register systems with RMT/SMT.
 
 ## IMPLEMENTATION
-  SUSEConnect is implemented in Ruby 2.0.  It communicates with the registration
+  SUSEConnect is implemented in Ruby. It communicates with the registration
   server using a RESTful JSON API over HTTP using TLS encryption.
 
 ## ENVIRONMENT
-  SUSEConnect respects the HTTP_PROXY and HTTPS_PROXY environment variables.
+  SUSEConnect uses Net::HTTP for its requests and respects the HTTP_PROXY environment variable.
```

Comment:
Do you think it's necessary to mention "uses Net::HTTP for its requests"? It seems more like an implementation detail to me, not sure if it's relevant for users.